src/Extension/FluentExtension.php

```diff
@@ -642,7 +573,7 @@ public function getLocalisedTable($tableName, $locale = '')
      * @param string $locale If passed, this is the locale we wish to delete in. If empty this is the root table
      * @return string
      */
-    protected function getDeleteTableTarget($tableName, $locale = '')
+    public function getDeleteTableTarget($tableName, $locale = '')
```

Comment:
This is a breaking change - any way we can avoid that?

Reply:
Hm yeah I can put it into a public accessor and call that, instead of changing the visibility. :)